src/lib/api/resources/AccountAccesses.ts

```diff
@@ -0,0 +1,61 @@
+import { AxiosInstance } from "axios";
+
+import CONFIG from "../../../config";
+
+import {
+  AccountAccessFilters,
+  DeleteAccountAccessResponse,
+} from "../../../types/api/account-accesses";
+import { Resource } from "../../../types/api/permissions";
+
+const { CLIENT_SETTINGS } = CONFIG;
+const { GENERAL_ENDPOINT } = CLIENT_SETTINGS;
+
+export default class AccountsAccessesApi {
+  private client: AxiosInstance;
+
+  private accountId?: number;
+
+  private accountAccessesURL: string;
+
+  constructor(client: AxiosInstance, accountId?: number) {
+    this.client = client;
+    this.accountId = accountId;
+    this.accountAccessesURL = `${GENERAL_ENDPOINT}/api/accounts/${this.accountId}/account_accesses`;
+  }
+
+  /**
+   * Get list of account accesses for which specifier_type is User or Invite.
+   * You have to have account admin/owner permissions for this endpoint to work.
+   * If you specify project_ids, inbox_ids or domain_uuids, the endpoint will return account accesses for these resources.
+   */
+  public async listUserAndInviteAccountAccesses(
```

Comment:
Let's maybe not specify implementation details for this route
- `listAccountAccesses` should be enough, we might start returning not only Users and Invites :)